packages/pigeon/bin/pigeon.dart

```diff
@@ -13,20 +15,28 @@ Future<void> main(List<String> args) async {
       (opts.input != null) ? 'import \'${opts.input}\';\n' : '';
   final String code = """$importLine
 import 'dart:io';
+import 'dart:isolate';
 import 'package:pigeon/pigeon_lib.dart';
 
-void main(List<String> args) async {
-  exit(await Pigeon.run(args));
+void main(List<String> args, SendPort sendPort) async {
+  sendPort.send(await Pigeon.run(args));
 }
 """;
   // TODO(aaclarke): Start using a system temp file.
```

Comment:
Just use https://api.dart.dev/stable/2.9.3/dart-io/Directory/systemTemp.html?

Reply:
Yea, that API isn't so helpful though because I should probably make a temp directory in there instead of placing the file directly in the system temp.  It would be nice if we had something like `mktemp -d`.  At the very least i need a GUID to make sure I don't collide.  I'll get around to this later.

Reply:
Oops sorry wrong link. I meant to point to https://api.dart.dev/stable/2.9.3/dart-io/Directory/createTempSync.html

Reply:
Ok, dang you, I did it, haha.  I had to do some trickery to get relative paths for the dart imports.  Now it says stuff like:
```dart
import '../../../../../../Users/aaclarke/dev/packages/packages/pigeon/pigeons/message.dart';
```

I didn't check to see if dart actually supports absolute path imports.

Reply:
Would this whole problem go away once we do https://github.com/flutter/flutter/issues/65071?

Reply:
I don't think so.  There always has to be a link between the generated code that the isolate executes and the pigeon file.